tests/api/v1/endpoints/test_oauth_endpoints.py

```diff
@@ -390,6 +390,28 @@ def test_invalid_client_secret(self, db, url, api_client):
 
         new_client.delete(db)
 
+    def test_get_access_token_root_client(self, url, api_client):
+        data = {
+            "client_id": config.security.OAUTH_ROOT_CLIENT_ID,
+            "client_secret": config.security.OAUTH_ROOT_CLIENT_SECRET,
+        }
+
+        response = api_client.post(url, data=data)
+        jwt = json.loads(response.text).get("access_token")
+        assert 200 == response.status_code
+        assert (
+            data["client_id"]
+            == json.loads(extract_payload(jwt, config.security.APP_ENCRYPTION_KEY))[
+                JWE_PAYLOAD_CLIENT_ID
+            ]
+        )
+        assert (
+            json.loads(extract_payload(jwt, config.security.APP_ENCRYPTION_KEY))[
+                JWE_PAYLOAD_SCOPES
+            ]
+            is None
```

Comment:
sorry i'm probably just missing something, why are there no scopes here? I thought the root client got all the scopes.

Reply:
I didn't see anywhere in the original fidesops root client where that happed. What I saw it looked like it got set to `None`. Maybe I missed it somewhere?